mcp/strategy.go

```
@@ -0,0 +1,119 @@
+				Date:   dateArray[i],
+				Open:   data.Opening[i],
+				Close:  data.Closing[i],
+				High:   data.High[i],
+				Low:    data.Low[i],
+				Volume: data.Volume[i],
+			}
+		}
+	}()
+
+	repository := asset.NewInMemoryRepository()
+	repository.Append("in_memory_asset", snapshots)
```

golangci-lint (1.64.8)

Error return value of `repository.Append` is not checked

(errcheck)